cpp/cmake/thirdparty/patches/nanoarrow_cmake.diff

```diff
@@ -33,7 +33,7 @@ index 8714c70..1feec13 100644
 
 -  install(TARGETS nanoarrow DESTINATION lib)
 +  install(TARGETS nanoarrow
-+          DESTINATION lib
++          DESTINATION "${CMAKE_INSTALL_LIBDIR}"
 +          EXPORT nanoarrow-exports)
    install(DIRECTORY src/
            DESTINATION include
```

Comment:
While we're at it, should we also patch this to `CMAKE_INSTALL_INCLUDEDIR`?

Reply:
Yes we should.

Reply:
Ugh sorry and I missed one more. There's another reference to include in the line below where nanoarrow_config.h is installed.

Reply:
I pushed a fully updated set of patches. Let me know if everything looks OK.

Reply:
No it doesn't look OK. There seems to be some formatting issue with it (https://github.com/rapidsai/cudf/pull/15456/commits/f5dc004b0748fd3af691c48ffbe19f414c0acac3).

Reply:
Didn't realize git diff would add trailing whitespaces. That's fixed now.